### ISSUE TITLE
Nuvola crashes when Close button in activation dialog is pressed after activation

### ISSUE THREAD

**Issue by fenryxo** [tiliado/nuvolaruntime#448]
Steps to reproduce:
- Go through the activation procedure
- Click popover close button after activation.
- Nuvola crashes.

Notes:
- Cannot reproduce in Nuvola CDK and Nuvola flatpak on Debian Stretch.
- Need to get a backtrace.

**Comment by fenryxo** [comment]
Backtrace:

```
Core was generated by `nuvolaruntime -a /app/share/nuvolaruntime/web_apps/nuvola_demo_player -D'.
Program terminated with signal SIGTRAP, Trace/breakpoint trap.
#0  _g_log_abort (breakpoint=breakpoint@entry=1) at gmessages.c:583
583	}
[Current thread is 1 (Thread 0x7f5033612dc0 (LWP 110))]
warning: File "/usr/lib/libstdc++.so.6.0.22-gdb.py" auto-loading has been declined by your `auto-load safe-path' set to "$debugdir:$datadir/auto-load".
To enable execution of this file add
	add-auto-load-safe-path /usr/lib/libstdc++.so.6.0.22-gdb.py
line to your configuration file "/home/fenryxo/.gdbinit".
To completely disable this security protection add
	set auto-load safe-path /
line to your configuration file "/home/fenryxo/.gdbinit".
For more information about this security protection see the
"Auto-loading safe path" section in the GDB manual.  E.g., run from the shell:
	info "(gdb)Auto-loading safe path"
(gdb) bt
#0  0x00007f505452f541 in _g_log_abort (breakpoint=breakpoint@entry=1) at gmessages.c:583
#1  0x00007f505453083d in g_logv (log_domain=0x7f5054830464 "GLib-GObject", log_level=G_LOG_LEVEL_CRITICAL, format=<optimized out>, args=args@entry=0x7ffcdf1d6c60) at gmessages.c:1391
#2  0x00007f505453099f in g_log (log_domain=log_domain@entry=0x7f5054830464 "GLib-GObject", log_level=log_level@entry=G_LOG_LEVEL_CRITICAL, format=format@entry=0x7f505457c947 "%s: assertion '%s' failed") at gmessages.c:1432
#3  0x00007f50545311c9 in g_return_if_fail_warning (log_domain=log_domain@entry=0x7f5054830464 "GLib-GObject", pretty_function=pretty_function@entry=0x7f50548338d8 <__func__.14602> "g_object_ref", expression=expression@entry=0x7f505483269c "G_IS_OBJECT (object)")
    at gmessages.c:2809
#4  0x00007f50548090bc in g_object_ref (_object=_object@entry=0x2015600) at gobject.c:3206
#5  0x00007f50530260be in gtk_grab_notify_foreach (child=0x2015600, data=0x7ffcdf1d6f10) at gtkmain.c:2129
#6  0x00007f5052fe3610 in gtk_grid_forall (container=<optimized out>, include_internals=<optimized out>, callback=0x7f5053025e30 <gtk_grab_notify_foreach>, callback_data=0x7ffcdf1d6f10) at gtkgrid.c:542
#7  0x00007f50530260a2 in gtk_grab_notify_foreach (child=0x20fb5a0 [NuvolaTiliadoTrialWidget], data=0x7ffcdf1d6f10)
    at gtkmain.c:2132
#8  0x00007f5052fe3610 in gtk_grid_forall (container=<optimized out>, include_internals=<optimized out>, callback=0x7f5053025e30 <gtk_grab_notify_foreach>, callback_data=0x7ffcdf1d6f10) at gtkgrid.c:542
#9  0x00007f50530260a2 in gtk_grab_notify_foreach (child=0x210e420 [GtkGrid], data=0x7ffcdf1d6f10) at gtkmain.c:2132
#10 0x00007f505318b969 in gtk_window_forall (container=0x2366330 [NuvolaWebAppWindow], include_internals=1, callback=0x7f5053025e30 <gtk_grab_notify_foreach>, callback_data=0x7ffcdf1d6f10) at gtkwindow.c:8534
#11 0x00007f50530260a2 in gtk_grab_notify_foreach (child=child@entry=0x2366330 [NuvolaWebAppWindow], data=data@entry=0x7ffcdf1d6f10)
    at gtkmain.c:2132
#12 0x00007f50530261bb in gtk_grab_notify (group=group@entry=0x2340f40 [GtkWindowGroup], device=device@entry=0x0, old_grab_widget=old_grab_widget@entry=0x1ec4430 [GtkPopover], new_grab_widget=<optimized out>, from_grab=from_grab@entry=0) at gtkmain.c:2205
#13 0x00007f5053026fdb in gtk_grab_remove (widget=0x1ec4430 [GtkPopover]) at gtkmain.c:2293
#14 0x00007f5053084c14 in gtk_popover_apply_modality (popover=0x1ec4430 [GtkPopover], modal=<optimized out>) at gtkpopover.c:536
#15 0x00007f5053085b32 in gtk_popover_hide_internal (popover=0x1ec4430 [GtkPopover]) at gtkpopover.c:319
#16 0x00007f5053085b50 in gtk_popover_hide (widget=0x1ec4430 [GtkPopover]) at gtkpopover.c:1661
#20 0x00007f505481fb1f in <emit signal ??? on instance 0x1ec4430 [GtkPopover]> (instance=instance@entry=0x1ec4430, signal_id=<optimized out>, detail=detail@entry=0) at gsignal.c:3447
    #17 0x00007f5054804475 in g_closure_invoke (closure=closure@entry=0x1ed72d0, return_value=return_value@entry=0x0, n_param_values=1, param_values=param_values@entry=0x7ffcdf1d71c0, invocation_hint=invocation_hint@entry=0x7ffcdf1d7140) at gclosure.c:804
    #18 0x00007f5054816633 in signal_emit_unlocked_R (node=node@entry=0x1ed7320, detail=detail@entry=0, instance=instance@entry=0x1ec---Type <return> to continue, or q <return> to quit---
4430, emission_return=emission_return@entry=0x0, instance_and_params=instance_and_params@entry=0x7ffcdf1d71c0) at gsignal.c:3565
    #19 0x00007f505481f738 in g_signal_emit_valist (instance=<optimized out>, signal_id=<optimized out>, detail=<optimized out>, var_args=var_args@entry=0x7ffcdf1d7370) at gsignal.c:3391
#21 0x00007f505317f3f9 in gtk_widget_hide (widget=0x1ec4430 [GtkPopover]) at gtkwidget.c:4901
#22 0x00007f50548046a4 in _g_closure_invoke_va (closure=closure@entry=0x2667840, return_value=return_value@entry=0x0, instance=instance@entry=0x2663340, args=args@entry=0x7ffcdf1d7680, n_params=<optimized out>, param_types=0x0) at gclosure.c:867
#23 0x00007f505481f425 in g_signal_emit_valist (instance=0x2663340, signal_id=<optimized out>, detail=0, var_args=var_args@entry=0x7ffcdf1d7680) at gsignal.c:3300
#24 0x00007f505481fb1f in g_signal_emit (instance=<optimized out>, signal_id=<optimized out>, detail=<optimized out>)
    at gsignal.c:3447
#25 0x00007f5052f230dd in gtk_button_do_release (button=0x2663340 [GtkButton], emit_clicked=<optimized out>) at gtkbutton.c:1843
#26 0x00007f5052f23145 in gtk_real_button_released (button=0x2663340 [GtkButton]) at gtkbutton.c:1961
#30 0x00007f505481fb1f in <emit signal ??? on instance 0x2663340 [GtkButton]> (instance=instance@entry=0x2663340, signal_id=<optimized out>, detail=detail@entry=0) at gsignal.c:3447
    #27 0x00007f5054804475 in g_closure_invoke (closure=closure@entry=0x1f22260, return_value=return_value@entry=0x0, n_param_values=1, param_values=param_values@entry=0x7ffcdf1d7970, invocation_hint=invocation_hint@entry=0x7ffcdf1d78f0) at gclosure.c:804
    #28 0x00007f5054816633 in signal_emit_unlocked_R (node=node@entry=0x1f6edf0, detail=detail@entry=0, instance=instance@entry=0x2663340, emission_return=emission_return@entry=0x0, instance_and_params=instance_and_params@entry=0x7ffcdf1d7970) at gsignal.c:3565
    #29 0x00007f505481f738 in g_signal_emit_valist (instance=<optimized out>, signal_id=<optimized out>, detail=<optimized out>, var_args=var_args@entry=0x7ffcdf1d7b20) at gsignal.c:3391
#31 0x00007f5052f215a0 in multipress_released_cb (gesture=0x2238a90 [GtkGestureMultiPress], n_press=<optimized out>, x=<optimized out>, y=<optimized out>, widget=0x2663340 [GtkButton]) at gtkbutton.c:666
#32 0x0000003133405ed8 in ffi_call_unix64 () at ../libffi-3.2.1/src/x86/unix64.S:76
#33 0x000000313340593a in ffi_call (cif=cif@entry=0x7ffcdf1d7e80, fn=fn@entry=0x7f5052f21580 <multipress_released_cb>, rvalue=<optimized out>, avalue=avalue@entry=0x7ffcdf1d7d50) at /usr/src/debug/libffi/3.2.1-r0/libffi-3.2.1/src/x86/ffi64.c:525
#34 0x00007f505480518a in g_cclosure_marshal_generic_va (closure=0x2667340, return_value=0x0, instance=<optimized out>, args_list=<optimized out>, marshal_data=<optimized out>, n_params=3, param_types=0x20c0f50) at gclosure.c:1604
#35 0x00007f50548046a4 in _g_closure_invoke_va (closure=closure@entry=0x2667340, return_value=return_value@entry=0x0, instance=instance@entry=0x2238a90, args=args@entry=0x7ffcdf1d8110, n_params=<optimized out>, param_types=0x20c0f50) at gclosure.c:867
#36 0x00007f505481f425 in g_signal_emit_valist (instance=0x2238a90, signal_id=<optimized out>, detail=0, var_args=var_args@entry=0x7ffcdf1d8110) at gsignal.c:3300
#37 0x00007f505481fb1f in g_signal_emit (instance=instance@entry=0x2238a90, signal_id=<optimized out>, detail=detail@entry=0)
---Type <return> to continue, or q <return> to quit---
    at gsignal.c:3447
#38 0x00007f5052fdf106 in gtk_gesture_multi_press_end (gesture=0x2238a90 [GtkGestureMultiPress], sequence=<optimized out>)
    at gtkgesturemultipress.c:283
#39 0x00007f5054807427 in g_cclosure_marshal_VOID__BOXEDv (closure=0x1fce7e0, return_value=<optimized out>, instance=0x2238a90, args=<optimized out>, marshal_data=<optimized out>, n_params=<optimized out>, param_types=0x1fce810) at gmarshal.c:1950
#40 0x00007f50548046a4 in _g_closure_invoke_va (closure=closure@entry=0x1fce7e0, return_value=return_value@entry=0x0, instance=instance@entry=0x2238a90, args=args@entry=0x7ffcdf1d84a0, n_params=<optimized out>, param_types=0x1fce810) at gclosure.c:867
#41 0x00007f505481f425 in g_signal_emit_valist (instance=0x2238a90, signal_id=<optimized out>, detail=0, var_args=var_args@entry=0x7ffcdf1d84a0) at gsignal.c:3300
#42 0x00007f505481fb1f in g_signal_emit (instance=instance@entry=0x2238a90, signal_id=<optimized out>, detail=detail@entry=0)
    at gsignal.c:3447
#43 0x00007f5052fdc3de in _gtk_gesture_check_recognized (sequence=0x0, recognized=0, gesture=0x2238a90 [GtkGestureMultiPress])
    at gtkgesture.c:345
#44 0x00007f5052fdc3de in _gtk_gesture_check_recognized (gesture=gesture@entry=0x2238a90 [GtkGestureMultiPress], sequence=sequence@entry=0x0) at gtkgesture.c:386
#45 0x00007f5052fdd9eb in gtk_gesture_handle_event (controller=0x2238a90 [GtkGestureMultiPress], event=0x292f8a0)
    at gtkgesture.c:777
#46 0x00007f5052fe066d in gtk_gesture_single_handle_event (controller=0x2238a90 [GtkGestureMultiPress], event=0x292f8a0)
    at gtkgesturesingle.c:222
#47 0x00007f5052fad8be in gtk_event_controller_handle_event (controller=0x2238a90 [GtkGestureMultiPress], event=event@entry=0x292f8a0) at gtkeventcontroller.c:230
#48 0x00007f50531741eb in _gtk_widget_run_controllers (widget=0x2663340 [GtkButton], event=0x292f8a0, phase=GTK_PHASE_BUBBLE)
    at gtkwidget.c:7379
#49 0x00007f5053028b91 in _gtk_marshal_BOOLEAN__BOXEDv (closure=0x1e5d4d0, return_value=0x7ffcdf1d8850, instance=0x2663340, args=<optimized out>, marshal_data=<optimized out>, n_params=<optimized out>, param_types=0x1e5d500) at gtkmarshalers.c:130
#50 0x00007f50548046a4 in _g_closure_invoke_va (closure=closure@entry=0x1e5d4d0, return_value=return_value@entry=0x7ffcdf1d8850, instance=instance@entry=0x2663340, args=args@entry=0x7ffcdf1d8920, n_params=<optimized out>, param_types=0x1e5d500) at gclosure.c:867
#51 0x00007f505481eef1 in g_signal_emit_valist (instance=0x2663340, signal_id=<optimized out>, detail=0, var_args=var_args@entry=0x7ffcdf1d8920) at gsignal.c:3300
#52 0x00007f505481fb1f in g_signal_emit (instance=instance@entry=0x2663340, signal_id=<optimized out>, detail=detail@entry=0)
    at gsignal.c:3447
#53 0x00007f50531764cc in gtk_widget_event_internal (widget=0x2663340 [GtkButton], event=0x292f8a0) at gtkwidget.c:7744
---Type <return> to continue, or q <return> to quit---
#54 0x00007f5053025b0e in propagate_event (topmost=<optimized out>, event=<optimized out>, widget=0x2663340 [GtkButton])
    at gtkmain.c:2582
#55 0x00007f5053025b0e in propagate_event (widget=<optimized out>, event=0x292f8a0, captured=<optimized out>, topmost=0x0)
    at gtkmain.c:2684
#56 0x00007f5053027bde in gtk_main_do_event (event=0x292f8a0) at gtkmain.c:1915
#57 0x00007f5052b41dd5 in _gdk_event_emit (event=event@entry=0x292f8a0) at gdkevents.c:73
#58 0x00007f5052b73072 in gdk_event_source_dispatch (source=<optimized out>, callback=<optimized out>, user_data=<optimized out>)
    at gdkeventsource.c:367
#59 0x00007f5054529ab7 in g_main_context_dispatch (context=0x1e74d00) at gmain.c:3177
#60 0x00007f5054529ab7 in g_main_context_dispatch (context=context@entry=0x1e74d00) at gmain.c:3830
#61 0x00007f5054529d28 in g_main_context_iterate (context=context@entry=0x1e74d00, block=block@entry=0, dispatch=dispatch@entry=1, self=<optimized out>) at gmain.c:3903
#62 0x00007f5054529ddc in g_main_context_iteration (context=0x1e74d00, may_block=0) at gmain.c:3964
#63 0x00007f504168fd33 in base::MessagePumpGlib::Run(base::MessagePump::Delegate*) ()
    at ../../base/message_loop/message_pump_glib.cc:305
#64 0x00007f50416ad4c4 in Run() () at ../../base/run_loop.cc:130
#65 0x00007f504152384f in CefBrowserMessageLoop::RunMessageLoop() () at ../../cef/libcef/browser/browser_message_loop.cc:121
#66 0x00007f505504a228 in nuvola_app_runner_controller_real_run_mainloop (base=0x20ee230 [NuvolaAppRunnerController])
    at src/nuvolakit-runner/AppRunnerController.c:4716
#67 0x00007f505504a03d in ___lambda65__gsource_func (self=<optimized out>) at src/nuvolakit-runner/AppRunnerController.c:4655
#68 0x00007f505504a03d in ___lambda65__gsource_func (self=<optimized out>) at src/nuvolakit-runner/AppRunnerController.c:4665
#69 0x00007f505452996a in g_main_context_dispatch (context=0x1e74d00) at gmain.c:3177
#70 0x00007f505452996a in g_main_context_dispatch (context=context@entry=0x1e74d00) at gmain.c:3830
#71 0x00007f5054529d28 in g_main_context_iterate (context=context@entry=0x1e74d00, block=block@entry=1, dispatch=dispatch@entry=1, self=<optimized out>) at gmain.c:3903
#72 0x00007f5054529ddc in g_main_context_iteration (context=context@entry=0x1e74d00, may_block=may_block@entry=1) at gmain.c:3964
#73 0x00007f5054ae77ad in g_application_run (application=application@entry=0x20ee230 [NuvolaAppRunnerController], argc=argc@entry=1, argv=argv@entry=0x7ffcdf1d9038) at gapplication.c:2470
#74 0x00007f50550aabcb in nuvola_startup_run_web_app_with_dbus_handshake (web_app_dir=web_app_dir@entry=0x1e608e0, argv=0x7ffcdf1d9038, argv_length1=1, error=error@entry=0x7ffcdf1d8f38) at src/nuvolakit-runner/Startup.c:338
#75 0x0000000000401316 in nuvola_main (args=<optimized out>, args_length1=<optimized out>) at src/apprunner/apprunner.c:224
#76 0x0000003130820291 in __libc_start_main (main=
```

**Comment by fenryxo** [comment]
Released with **Nuvola 4.11.60**, more info: [Nuvola Brings Redesigned Preferences Dialog, More Appearance Settings, and Bug Fixes](https://medium.com/nuvola-news/nuvola-brings-redesigned-preferences-dialog-more-appearance-settings-and-bug-fixes-6545325b35f3)
